package.json

```diff
@@ -27,7 +27,10 @@
     "start": "node src/backend",
     "server": "node src/backend/web/server",
     "hint": "hint http://localhost:3000",
-    "webhint": "cross-env PORT=3000 npm-run-all -r -p server hint"
+    "webhint": "cross-env PORT=3000 npm-run-all -r -p server hint",
+    "debug-server": "nodemon server.js",
```

Comment:
This was removed recently, compare to what's on `master` now: https://github.com/Seneca-CDOT/telescope/blob/master/package.json#L16-L30.  You should make sure you are in sync with `master`.